Makefile

```diff
@@ -1,23 +1,43 @@
-.PHONY: compose_build test_db create_database clean bundle tests build watch start
+.PHONY: compose_build up test_db create_database clean down bundle tests lint backend-unit-tests frontend-unit-tests test build watch start
 
 compose_build:
 	docker-compose build
 
+up:
+	docker-compose up -d --build
+
 test_db:
-	docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
+	docker-compose exec postgres sh -c 'psql -U postgres -c "drop database if exists tests;" && psql -U postgres -c "create database tests;"'
 
 create_database:
 	docker-compose run server create_db
 
 clean:
 	docker ps -a -q | xargs docker kill;docker ps -a -q | xargs docker rm
 
+down:
+	docker-compose down
+
 bundle:
 	docker-compose run server bin/bundle-extensions
 
 tests:
 	docker-compose run server tests
 
+lint:
+	./flake8_tests.sh
```

Comment:
Which reminds me, could we move this script into the `bin/` directory as all the other scripts?